google_service.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/user"
+	"path/filepath"
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2"
+)
+
+var clientIdFile = environmentName + "/client_id.json"
+
+func getClient(ctx context.Context, config *oauth2.Config) *http.Client {
+	cacheFile, err := tokenCacheFile()
+
+	if err != nil {
+		log.Fatalf("Unable to get path to cached credential file. %v", err)
+	}
+
+	tok, err := tokenFromFile(cacheFile)
+
+	if err != nil {
+		tok = getTokenFromWeb(config)
+		saveToken(cacheFile, tok)
+	}
+
+	return config.Client(ctx, tok)
+}
+
+func getTokenFromWeb(config *oauth2.Config) *oauth2.Token {
```

Comment:
bunlar Google'in orneklerinden.

Lisansina baktin mi? Istegimiz gibi kullanabiliyor muymusuz?